modern/src/map/core/useMapStyles.js

```diff
@@ -185,4 +187,11 @@ export default () => {
       available: !!customMapUrl,
     },
   ];
+
+  for (let i = 0; i < Styles.length; i += 1) {
+    if (!disabledStyles.has(Styles[i].id)) {
+      activeStyles.push(Styles[i]);
+    }
+  }
+  return activeStyles;
```

Comment:
Just use `styles.filter(...)` instead of doing this loop. And no need for a separate variable in that case.